autosubmit/autosubmit.py

```diff
@@ -1953,35 +1953,35 @@ def check_wrappers(
         :param expid: a string with the experiment id
         :return: non-wrapped jobs to check and a dictionary with the changes in the jobs status
         """
-        jobs_to_check: dict[str, list[list[Job]]] = dict()
+        jobs_to_check: dict[str, list[list[Job]]] = defaultdict(list)
```

Comment:
We had the pattern

```py
if v not in dic:
  dic[v] = []
else:
  dic.append(v)
```

so replaced that by the defaultdict(list) here so we don't need that if any longer.